_migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md

```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
 
 # IAM and security groups for existing clusters
 
-This page outlines scenarios for using the migration tools with existing clusters, including any necessary configuration changes to ensure proper communication between them.
+This page outlines security scenarios for using the migration tools with existing clusters, including any necessary configuration changes to ensure proper communication between them.
 
-## Importing an OpenSearch Service or OpenSearch Serverless Target Cluster
+## Importing an OpenSearch Service or OpenSearch Serverless target cluster
 
-### OpenSearch Service
+Use the following scenarios for OpenSearch Service or Serverless target clusters:
+
+### OpenSearch service
 
 For an OpenSearch Domain, two main configurations are typically required to ensure proper functioning of the migration solution:
 
-1. **Security Group Configuration**:  
-   The Domain should have a security group that allows communication from the applicable Migration services (Traffic Replayer, Migration Console, Reindex-from-Snapshot). The CDK will automatically create an `osClusterAccessSG` security group, which is applied to the Migration services. The user should then add this security group to their existing Domain to allow access.
+1. **Security Group Configuration**
+
```

Comment:
To the best of my knowledge, "domain" is not capitalized as a proper noun. Please lowercase across this and the other files.